tests/director/external-services-mock-integration/async_api_delete_test.go

```diff
@@ -0,0 +1,171 @@
+package external_services_mock_integration
```

Comment:
Not sure these tests should be in the `tests/director/external-service-mock-intergration` directory. We created the [`tests/operations-controller`](https://github.com/kyma-incubator/compass/tree/master/tests/operations-controller) directory for precisely these tests.
From one point of view we do test scenarios relating to external systems in general in the `tests/director/external-service-mock-intergration` directory (ex. auditlog & fetch requests) and in this sense these tests do make sense to be here.
But from another point of view these tests cover much bigger scenarios than the auditlog & fetch requests scenarios. Also eventually we might want to add more operations-controller e2e tests which would potentially make the `tests/director/external-service-mock-intergration` directory very operations-controller -centric instead of having more general purpose tests in it which don't have a better place to be in. For these exact reasons I think we wanted to have the `tests/operations-controller` directory.

Maybe we can discuss it in the next daily standup? 🙂

Reply:
Sure, let discuss it.